projects/tsq-gallery/src/lib/components/main-tsq-gallery/main-tsq-gallery.component.ts

```diff
@@ -20,10 +21,15 @@ export class TSqGalleryComponent {
   /** Files to be displayed at the Preview and Viewer. Accepted types: images and pdfs. */
   @Input() files: TSqGalleryFileModel[];
 
+    /** Override the class of the container for each file of the Preview.
+     * The class should either have `::ng-deep` (not recommended) or be global.
+     */
+  @Input() containerClass: string;
```

Comment:
Is this Input still necessary?

Reply:
Yes, just the description is not updated; This one changes the container of images, while the other one, for each of then. I can show you a use case currently at the ADM, but I think there are better ways to do it still